airflow_ml_dags/dags/02_train_model.py

```diff
@@ -0,0 +1,114 @@
+from datetime import timedelta
+from airflow import DAG
+from airflow.providers.docker.operators.docker import DockerOperator
+from airflow.utils.dates import days_ago
+from docker.types import Mount
+
+
+default_args = {
+    'owner': 'airflow',
+    'email' : ['airflow@example.com'],
+    'retries': 1,
+    'retry_delay': timedelta(minutes=5),
+}
+
+with DAG(
+    '02_train_model',
+    default_args=default_args,
+    description='Download data. Preproccessing data. Splitting data. Train & Save model.',
+    schedule_interval="@weekly",
+    start_date=days_ago(2),
+) as dag:
+    
+
+    files = DockerOperator(
+        task_id='Download_data',
+        image='airflow-preprocess-files',
+        network_mode='bridge',
+        do_xcom_push=False,
+        mounts=[
+            Mount(
+                source='/home/d3vyatk4ru/Desktop/TP-ml-prod-Airflow/data',
+                target='/data',
+                type='bind',
+        )],
+        command='--in_path /data/raw/{{ ds }} --tmp_path /data/tmp/{{ ds }}',
+    )
+
+    preprocessing = DockerOperator(
+        task_id='Preprocessing_features',
+        image='airflow-preprocessing',
+        network_mode='bridge',
+        do_xcom_push=False,
+        mounts=[
+            Mount(
+                source='/home/d3vyatk4ru/Desktop/TP-ml-prod-Airflow/data',
```

Comment:
https://stackoverflow.com/questions/3430372/how-do-i-get-the-full-path-of-the-current-files-directory